en/competition/index.md

```diff
@@ -7,34 +7,34 @@ ref: competition
 image: "/banner/competition.jpg"
 ---
 
-After participating for many years to the Walking Machine SAE and the Eurobot subsequently, since 2016, we participate to the Robocup@Home international competition.
+After participating for many years in the Walking Machine SAE and the Eurobot competitions subsequently, since 2016, we are competing in the Robocup@Home international competition.
 
-## Robocup@Home in brief
+## Robocup@Home in a few lines
 
-The RoboCup@Home competition is held during the Robocup, the most important competition of Robotics in the world. The latter also presents the Soccer, Rescue and Junior categories. This event held once a year, brings together university and college from across the world.
+The RoboCup@Home is held during the Robocup, the most important robotics competition in the world. The latter also showcase the Soccer, Rescue, and Junior categories. This event, held once a year, brings together university and college from across the world.
 
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/YpjeNa8BAYg" frameborder="0" allowfullscreen></iframe></center><br>
-RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in the tasks of everyday life. To determine the best robots, the teams move to their robot a series of tests to measure their skills and performance in an environment simulating a House. The main axes of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, object manipulation and artificial intelligence.
+RoboCup@Home aims to develop robots for personal assistance with the aim of creating the perfect assistant in everyday tasks. To find the best robots, the teams make their robot do a series of tests to measure their skills and performance in an house like environment. The main fields of development are: Human-machine interaction and cooperation, navigation and mapping in dynamic environment, computer vision and recognition of objects under different light, objects manipulation and artificial intelligence.
```

Comment:
- measure their skills and performance in an domestic environment.
 - human-machine interaction - human en minuscule
 - dynamic environmental navigation and mapping
 - object recognition